tests/market/test_future.py

```diff
@@ -43,10 +46,12 @@ def active_future_market() -> FutureMarkets:
             grid_fees=GridFee(grid_fee_percentage=area.grid_fee_percentage,
                               grid_fee_const=area.grid_fee_constant),
             name=area.name)
-    future_market.create_future_markets(DEFAULT_CURRENT_MARKET_SLOT, DEFAULT_SLOT_LENGTH)
+    future_market.create_future_markets(
+        DEFAULT_CURRENT_MARKET_SLOT, DEFAULT_SLOT_LENGTH, area.config)
     yield future_market
 
     GlobalConfig.future_market_duration = orig_future_market_duration
+    GlobalConfig.start_date = orig_start_date
```

Comment:
I think you do not need that because you are not overwriting the `GlobalConfig.start_date` earlier  but only the `area.config.start_date`.

Reply:
Unfortunately if I do not revert it followup tests fail. Looks like the reference is updated and not the value of the start_date.

Reply:
ok.